src/__tests__/mzData.js

```diff
@@ -0,0 +1,15 @@
+import {mzData} from '..';
+import {readFileSync} from 'fs';
+const pathFiles = __dirname + '/files/';
+
+describe('mzData', () => {
+    it('read tiny.mzData.xml', () => {
+        const data = readFileSync(pathFiles + 'tiny.mzData.xml');
+        var response = mzData(data);
+        expect(response.kind).toBe('mzData');
```

Comment:
ok indeed, the package is not meant to do an evaluation of this, so I can remove it